AirportDepartures.playground/Contents.swift

```diff
@@ -81,11 +81,11 @@ struct Flight {
 //: d. Make one of the flights have a `nil` terminal because it has not been decided yet.
 //:
 //: e. Stretch: Look at the API for [`DateComponents`](https://developer.apple.com/documentation/foundation/datecomponents?language=objc) for creating a specific time
-let myDeparture = DepartureBoard(destination: "West Palm Beach", airline: "Turkish Airlines", flights: [], departureTime: "06:00am", terminal: 5, status: .route )
+let myDeparture = DepartureBoard(destination: "West Palm Beach", airline: "Turkish Airlines", departureTime: "6:00am", terminal: 5, status: .route, flights: [] )
 
-let flight1 = Flight(pilots: 2, attendants: 2, passengers: 200, ticketClass: "Economy")
-let flight2 = Flight(pilots: 2, attendants: 4, passengers: 100, ticketClass: "Business")
-let flight3 = Flight(pilots: 2, attendants: 4, passengers: 20, ticketClass: "First Class")
+var flight1 = Flight(pilots: 2, attendants: 2, passengers: 200, ticketClass: "Economy")
+var flight2 = Flight(pilots: 2, attendants: 4, passengers: 100, ticketClass: "Business")
+var flight3 = Flight(pilots: 2, attendants: 4, passengers: 20, ticketClass: "First Class")
 
 myDeparture.flights.append(flight1)
```

Comment:
This is a great way to append in to your class without creating a new variable.  Good job.